docs/pages/2020_Intro_Week/sections/appendix_computer_code_setup.rst

```diff
@@ -0,0 +1,10 @@
+*These appendices consist of optional exercises, and are mentioned in earlier parts of the tutorial.
+Go through them only if you have time.*
+
+.. _2020:appendix_computer_code_setup:
+
+Setting up computers and codes
+==============================
+
+To setup a computer, please refer to the `online documentation <https://aiida-core.readthedocs.io/en/latest/get_started/computers.html#setup-a-computer>`_.
```

Comment:
this link no longer exists, we should also use inter-sphinx type references, which will pick up such issues at build time

Reply:
What do you mean inter-sphinx type reference, how does that work? Because this are two different repos, right? (the tutorial referencing the docs)

Reply:
yep thats perfectly fine, we use [intersphinx](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html) lo link between documentations 😄 Happy to explain how this works whenever), but basically just add the `:ref:` as you would in aiida-core, but append `aiida:`